### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by doron-cohen** [hasura/ra-data-hasura#63]
Hi!

I was wondering about the project's status. Is it maintained or being deprecated? Are there any plans to continue and develop features like Typescript support?

Thanks

**Comment by cpursley** [comment]
We're using it actively and using Typescript.

**Comment by praveenweb** [comment]
@doron-cohen - Your concerns are valid. It is definitely not deprecated. The original project `ra-data-hasura-graphql` was moved into this repo recently. We definitely want to keep the library up to date.

I do understand there are a few open issues that needs a quick fix, expect updates to keep happening every few weeks. 
Fundamentally the core library for react-admin is set and we need to add more support for Hasura specific stuff going forward.

**Comment by megantaylor** [comment]
following up on this, looks like quite a few fixes were merged into master since may, but the last release was in jan. any plans for an updated release soon?

**Comment by praveenweb** [comment]
@megantaylor - Hi, It is planned for later this week. Currently waiting on #65 to be merged.

**Comment by praveenweb** [comment]
We have just released a new version `v0.2.0`. Check the changelog [here](https://github.com/hasura/ra-data-hasura/releases/tag/v0.2.0).

Closing this issue since we want to keep this project active :)

**Comment by cpursley** [comment]
Thanks @praveenweb !
